psij-ci-run

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python tests/ci_runner.py
```

Comment:
this should be `python3`.  On my system:
```sh
$ python -V
Python 2.7.17
```

Reply:
Ah, thanks! Yes, this should follow the same rules as above.

Reply:
Is it the same with pip? Aka, do I need to say `pip3` on your system to do the right thing?

Reply:
Yes, same with pip indeed.

Reply:
I updated this for both python and pip